dll/cpl/sysdm/lang/pt-PT.rc

```diff
@@ -72,9 +72,9 @@ CAPTION "Definições de sistema"
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX "Informações de versão", IDC_STATIC, 6, 3, 210, 73
-    LTEXT "O ReactOS é construído como servidor de sistemas operativos  e responde como tal. Marque esta opção para seleccionar apenas aplicativos.", IDC_STATIC, 15, 15, 183, 41
+    LTEXT "O ReactOS é construído como servidor de sistemas operativos  e responde como tal. Marque esta opção para aplicar isso apenas a aplicativos.", IDC_STATIC, 15, 15, 183, 41
```

Comment:
After `operativos` and before `e responde` (first sentence), there is an extra space too much.